assets/scss/main.scss

```diff
@@ -188,3 +191,80 @@ label.form-label[for="signer"] {
     }
   }
 }
+
+#date-picker {
+
```

Comment:
We should pull this out into its own date-picker.scss file otherwise main.scss will become unmanageable over time.